src/inverseRobot.js

```diff
@@ -7,7 +7,25 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
```

Comment:
The variable name 'inverse' is not very descriptive. Consider using a more descriptive name, such as 'invertedRobot'.